drivers/scheduler/k8s/k8s.go

```diff
@@ -1970,6 +2068,23 @@ func (k *K8s) IsScalable(spec interface{}) bool {
 	return false
 }
 
+// GetTokenFromConfigMap -  Retrieve the config map object and get auth-token
+func (k *K8s) GetTokenFromConfigMap(configMapName string) string {
+	var token string
+	k8sOps := k8s_ops.Instance()
+	if configMap, err := k8sOps.GetConfigMap(configMapName, "default"); err == nil {
+		secretName := configMap.Data["secret_name"]
```

Comment:
nit: no need to save into variables. Can you used from map directly for brevity.

Reply:
Fixed.